tools/stats/upload_stats_lib.py

```diff
@@ -199,6 +199,23 @@ def read_from_s3(
     return [json.loads(result) for result in results if result]
 
 
+def remove_nan_inf(old: Any) -> Any:
```

Comment:
Could CH load a null value? if it does, and if you want to add another small dependency `simplejson` here, this could work

```
import simplejson

d = {"k": float("inf")}
simplejson.dumps(d, ignore_nan=True)
/// return '{"k": null}' as the JSON string
```

Reply:
It should, but I also just discovered that casting inf to int and then to string gives inf (as a string), which is pretty nice, while null would be converted to the default value of 0